fuse_constraints/include/fuse_constraints/absolute_pose_2d_stamped_constraint.h

```diff
@@ -88,21 +98,21 @@ class AbsolutePose2DStampedConstraint : public fuse_core::Constraint
    *
    * Order is (x, y, yaw)
    */
-  const fuse_core::Vector3d& mean() const { return mean_; }
+  const fuse_core::VectorXd& mean() const { return mean_; }
 
   /**
    * @brief Read-only access to the square root information matrix.
    *
    * Order is (x, y, yaw)
    */
-  const fuse_core::Matrix3d& sqrtInformation() const { return sqrt_information_; }
+  const fuse_core::MatrixXd& sqrtInformation() const { return sqrt_information_; }
 
   /**
    * @brief Compute the measurement covariance matrix.
    *
    * Order is (x, y, yaw)
```

Comment:
Add a disclaimer along the lines of:
```suggestion
   * Order is (x, y, yaw). Note that the returned covariance matrix will be full sized and in the stated order. If only a partial covariance matrix was provided in the constructor, this covariance matrix may be a different size and in a different order than the constructor input.
```
Add something similar to `mean()`, and something less similar to `sqrtInformation()`. `sqrtInformation()` may return a non-square matrix; that's worth a mention.